src/components/MegaButton/MegaButton.vue

```diff
@@ -14,8 +14,8 @@
     <label
       :for="id"
       :class="[
-        'h-full',
-        'flex flex-column items-center rounded-lg my-5 mr-10 inline-block border-2 border-transparent ring-4 ring-transparent relative shadow-input cursor-pointer',
+        'h-full flex flex-column items-center relative inline-block',
```

Comment:
I believe 'flex flex-column items-center' may be unused - because inline-block overwrites them - did you try removing them? does anything change?

Reply:
is flex-column a class? I think it may be flex-col but still overwritten as inline-block comes after

Reply:
or we could keep 'flex items-center' instead of inline-block

Reply:
I think flex column is what is stacking that 'Hiding in the closet' label stacked on top of 'Voidcat'

Reply:
no, the banner is absolute, and flex-column was actually a dead class (the correct one is flex-col) - we removed it and nothing changes

Reply:
Ah, ok - sorry I was just guessing at this by looking at the screenshot above, wasn't actually looking at the code